### ISSUE TITLE
Getting 'ParticleSystemInspector' is inaccessible due to its protection level

### ISSUE THREAD

**Issue by ColorTwist** [mob-sakai/ParticleEffectForUGUI#71]
After deleting older version 2.3 on unity 2019.2.17f
Assets\Coffee\ParticleEffectForUGUI-upm\Scripts\Editor\UIParticleEditor.cs(244,9): error CS0122: 'ParticleSystemInspector' is inaccessible due to its protection level

**Comment by mob-sakai** [comment]
Hi @ColorTwist 

Yes, it occurs in Unity2019.1 or later.
I will fix it.

**Comment by mob-sakai** [comment]
Ref #70

**Comment by mob-sakai** [comment]
Hi @ColorTwist 
Please try `3.0.0-preview.3`.

**Comment by ColorTwist** [comment]
Thanks!
I'll note that last time I installed 3.0 I had to rebuild all my UI Particle scripts as they become blank, even after restoring to 2.3.

related to NaughtyAttributes asset I guess. not sure now what i did or what happened.
 Need to release soon so i am afraid to try now :) after release and backup I'll try it.

![Untitled-2](https://user-images.githubusercontent.com/23701561/74634437-84875180-516c-11ea-818d-a5431d9d1ce6.png)
